openfisca_core/holders/tests/test_helpers.py

```diff
@@ -0,0 +1,98 @@
+import pytest
+
+from openfisca_core import holders, periods, tools
+from openfisca_core.entities import Entity
+from openfisca_core.holders import Holder
+from openfisca_core.periods import Instant, Period
+from openfisca_core.populations import Population
+from openfisca_core.variables import Variable
+
+
+@pytest.fixture
+def people():
+    return Entity(
+        key = "person",
+        plural = "people",
+        label = "An individual member of a larger group.",
+        doc = "People have the particularity of not being someone else.",
+        )
+
+
+@pytest.fixture
+def Income(people):
+    return type(
+        "Income",
+        (Variable,), {
+            "value_type": float,
+            "entity": people
+            },
+        )
+
+
+@pytest.fixture
+def population(people):
+    population = Population(people)
+    population.count = 1
+    return population
+
+
+@pytest.mark.parametrize("dispatch_unit, definition_unit, values, expected", [
+    [periods.YEAR, periods.YEAR, [1.], [3.]],
+    [periods.YEAR, periods.MONTH, [1.], [36.]],
+    [periods.YEAR, periods.DAY, [1.], [1096.]],
```

Comment:
Why it is not 1095 = 3 * 365 ? (Is it because a year is approx. 395.25 ?) and you rounded the number ?
It is different from the division counter-part.

Reply:
It is just because there's a leap year with 366 days.

I'm counting calendar days instead of multiplying by 365.